src/utils/metadata.ts

```diff
@@ -113,21 +129,24 @@ export function transformPublishFormToMetadata(
 ): MetadataMarket {
   const currentTime = toStringNoMS(new Date())
 
+  const transformedLinks = typeof links !== 'string' &&
```

Comment:
nitpick(non-blocking): It is basically the same check as above. Could this be generalized into one function? Or could at least both use the same style (both being a function).